pkg/virt-controller/services/template.go

```diff
@@ -1049,6 +1049,23 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 	selinuxType := t.clusterConfig.GetSELinuxLauncherType()
 	if selinuxType != virtconfig.DefaultSELinuxLauncherType {
```

Comment:
i know this isn't a part of the PR, but this `if` statement is a bit misleading. it took me awhile to grasp the intent behind what's going on.

the default selinuxlaunchertype is `""` at the moment, so really we're invoking this if statement if selinuxType is set to anything other than `""`.

Reply:
It took me a while too! Not sure what to do about that.

Reply:
`if selinuxType == ""`  might make it more obvious

Reply:
I can change it, but the opposite argument was made there: https://github.com/kubevirt/kubevirt/pull/3056/files#r376764854

Reply:
Well, all this does is checking if there is a policy to use. If we are not going to set the `virt-launcher.process` as default and I guess we shouldn't then I agree with David,  `if selinuxType == ""` is much more obvious